### PR TITLE
[zh] Update tasks\manage-kubernetes-objects

### PR DESCRIPTION
**Pull request by ydFu**
Update this page to upstream

list below:
```
1. content/zh-cn/docs/tasks/manage-kubernetes-objects/_index.md
2. content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md
```

### update description
1. Update out-of-sync content in English,miss the part of 'How to view an object' page.
2. Improve the description.
3. Add the anchor for link.
4. Fix the repeated word

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 1]
```diff
@@ -16,7 +17,8 @@ imperative commands built into the `kubectl` command-line tool. This document
 explains how those commands are organized and how to use them to manage live objects.
 -->
 使用构建在 `kubectl` 命令行工具中的指令式命令可以直接快速创建、更新和删除
-Kubernetes 对象。本文档解释这些命令的组织方式以及如何使用它们来管理现时对象。
+Kubernetes 对象。
+本文档解释这些命令的组织方式以及如何使用它们来管理现时对象。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md:21]:
```suggestion
本文档解释这些命令的组织方式以及如何使用它们来管理活跃对象。
```
`现时对象` has a very low hit when you search for via Google. May need to discuss further.
image

Comment (xichengliudui) [reply content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md:21]: `活跃对象` 相对比 `现时对象` 我认为更合适

Comment (ydFu) [reply content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md]: I also agree with this point of view.
Thank for @windsonsea and @xichengliudui.

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 2]
```diff
@@ -173,7 +175,7 @@ For more details on patch strings, see the patch section in
 用户对 Kubernetes 对象的模式定义有很好的了解：
 
 - `edit`：通过在编辑器中打开现时对象的配置，直接编辑其原始配置。
-- `patch`：通过使用补丁字符串（Patch String）直接更改某现时对象的的特定字段。
+- `patch`：通过使用补丁字符串（Patch String）直接更改某现时对象的特定字段。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md:178]:
```suggestion
- `patch`：通过使用补丁字符串（Patch String）直接更改某活跃对象的特定字段。
```
Do we need replace all the `现时对象`?

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 3]
```diff
@@ -16,7 +17,8 @@ imperative commands built into the `kubectl` command-line tool. This document
 explains how those commands are organized and how to use them to manage live objects.
 -->
 使用构建在 `kubectl` 命令行工具中的指令式命令可以直接快速创建、更新和删除
-Kubernetes 对象。本文档解释这些命令的组织方式以及如何使用它们来管理现时对象。
+Kubernetes 对象。
+本文档解释这些命令的组织方式以及如何使用它们来管理活跃对象。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md:21]: 没必要换行。

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 4]
```diff
@@ -191,34 +193,50 @@ You can use the `delete` command to delete an object from a cluster:
...
 
 {{< comment >}}
+<!---
 TODO(pwittrock): Uncomment this when implemented.
 
 You can use `kubectl view` to print specific fields of an object.
 
 - `view`: Prints the value of a specific field of an object.
+-->
+TODO(pwittrock)：实施时取消注释。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md:225]: 删除此行

File: content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md [hunk 5]
```diff
@@ -262,7 +277,7 @@ kubectl create service clusterip my-svc --clusterip="None" -o yaml --dry-run=cli
 You can use `kubectl create --edit` to make arbitrary changes to an object
 before it is created. Here's an example:
 -->
-## 在创建之前使用 `--edit` 更改对象
+## 在创建之前使用 `--edit` 更改对象 {##using-edit-to-modify-objects-before-creation}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/manage-kubernetes-objects/imperative-command.md:280]:
```suggestion
## 在创建之前使用 `--edit` 更改对象 {#using-edit-to-modify-objects-before-creation}
```

**Pull request merged**